app/campaigns/campaigns.controller.js

```diff
@@ -0,0 +1,23 @@
+const CampaignsService = require('./campaigns.service')
+
+const CampaignsController = {
+  getCampaigns
+}
+
+async function getCampaigns (request, response) {
+  const status = request.query.status
+  var campaigns = null
```

Comment:
I'd suggest using `let` instead of `var`. And you can also enforce that with a eslint rule